conjurapi/resource.go

```diff
@@ -75,6 +79,17 @@ func (c *Client) Resources(filter *ResourceFilter) (resources []map[string]inter
 	}
 
 	resources = make([]map[string]interface{}, 1)
-	err = json.Unmarshal(data, &resources)
+
+	// When count is enabled, the result is not an array
+	// so we have to add the result to an array to match the resulting type
+	if filter != nil {
```

Comment:
Currently with `filter == nil` this func returns empty resources, this isn't the expected behaviour.
The outer `if` isn't necessary.
```
if filter != nil && filter.Count {
	value := make(map[string]interface{})
	err = json.Unmarshal(data, &value)
	resources[0] = value
} else {
	err = json.Unmarshal(data, &resources)
}
```

Reply:
You are right, made the changes